.changeset/rare-bananas-add.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": minor
+---
+
+Use `esbuild-plugin-polyfill-node` for Node.js polyfill instead of `@esbuild-plugins` pack.
```

Comment:
Feels like we are fixing the broken aspects of the unsupported package?

```suggestion
fix: Use `esbuild-plugin-polyfill-node` for Node.js polyfill instead of `@esbuild-plugins` pack.
```

Reply:
The packages work, for now, actually. But they are out of date.